hooks/pre/32-install_selinux_packages.rb

```diff
@@ -1,5 +1,5 @@
 # Working around https://tickets.puppetlabs.com/browse/PUP-10548
-if facts.dig(:os, :selinux, :enabled)
+if facts[:os]['selinux']['enabled']
```

Comment:
I thought we symbolized all facts. I do think using `dig` is cleaner, but perhaps we should have some very early check (`pre_values`) that does roughly
```ruby
if facts.dig(:os, 'selinux', 'enabled').nil?
  fail('Unable to detect SELinux status')
end
```

Because facter should always set the boolean, even when it's disabled.

Reply:
In fact in just failed for me due to that reason. Trying with symbols now but I have a feeling it will also fail (as we believe it did with dig).

I might temporarily remove the condition entirely, just to prove to myself that this is where the issue lies, then come back and work out the correct syntax.